ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -187,25 +184,33 @@
   [context]
   (get-in context [:system :db]))
 
-(defn-timed get-collection-tasks
+(defn get-collection-tasks
   "Returns bulk update statuses with task ids by provider"
   [context provider-id]
   (get-provider-bulk-update-status (context->db context) provider-id))
 
+(declare get-bulk-update-task-status-for-provider)
+#_{:clj-kondo/ignore [:unresolved-symbol]}
 (defn-timed get-bulk-update-task-status-for-provider
   [context task-id provider-id]
   (get-bulk-update-task-status (context->db context) task-id provider-id))
 
+(declare get-bulk-update-collection-statuses-for-task)
+#_{:clj-kondo/ignore [:unresolved-symbol]}
 (defn-timed get-bulk-update-collection-statuses-for-task
   [context task-id]
   (get-bulk-update-task-collection-status (context->db context) task-id))
 
+(declare create-bulk-update-task)
+#_{:clj-kondo/ignore [:unresolved-symbol]}
 (defn-timed create-bulk-update-task
```

Comment:
Do we need get-bulk-update-task-status-for-provider, get-bulk-update-collection-statuses-for-task, create-bulk-update-task, update-bulk-update-task-collection-status all timed - can these be moved to debug?

Reply:
I would love to, I was trying to be very conservative. These must have had a middle of the road usage and time consumption.

Reply:
we talked on line, this is good as it is.